docs/structured-streaming-programming-guide.md

```diff
@@ -2119,7 +2119,7 @@ streamingDf.dropDuplicates("guid");
 // With watermark using guid and eventTime columns
 streamingDf
   .withWatermark("eventTime", "10 seconds")
-  .dropDuplicates("guid", "eventTime");
+  .dropDuplicates({"guid", "eventTime"});
```

Comment:
because of `@scala.annotation.varargs`, Java should work too with `dropDuplicates(col1: String, cols: String*)`?

Reply:
Same here.